app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
-    <string name="app_name">ParkingSystemJava</string>
+    <string name="app_name">FreeSpot</string>
+    <string name="main_activity_text_title">Welcome to Free Spot!</string>
+    <string name="main_activity_button_set_size">Select parking spaces</string>
 </resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done